services/gateway/shared/customMapper.js

```diff
@@ -32,6 +32,7 @@ const cast = {
 	base64: input => (input ? (Buffer.from(input)).toString('base64') : undefined),
 };
 
+// Safely retrieves value of a nested key inside an object. Return undefined if not found
```

Comment:
```suggestion
// Retrieves value of a nested key inside an object. Returns undefined when not found.
```

Also, could we please move the following method outside and add a couple of tests:
```
const test = (subObj, prop) => (subObj && subObj[prop] !== undefined);
```